### ISSUE TITLE
File missing text doesn't meet accessibility colour ratio because of markup

### ISSUE THREAD

**Issue by clarkepaul** [silverstripe/silverstripe-asset-admin#332]
The text just need to be moved up so it is nested within .gallery-item__thumbnail

Style change also needed for .gallery-item__error-message:
top: 50%;
z-index: 1;
![pasted_image_14_12_16__2_02_pm](https://cloud.githubusercontent.com/assets/555033/21165745/3edbcd1c-c206-11e6-901d-a997ded39d5e.png)

---

## Pull requests

- [ ] https://github.com/silverstripe/silverstripe-asset-admin/pull/421

**Comment by clarkepaul** [comment]
Easier solution is to add red bg color on gallery-item__error-message
